lighthouse-core/audits/dobetterweb/uses-optimized-images.js

```diff
@@ -89,21 +89,25 @@ class UsesOptimizedImages extends Audit {
       const url = URL.getDisplayName(image.url);
       const webpSavings = UsesOptimizedImages.computeSavings(image, 'webp');
 
-      let label = `${originalKb} KB total, webp savings: ${webpSavings.percent}%`;
       if (webpSavings.bytes > WEBP_ALREADY_OPTIMIZED_THRESHOLD_IN_BYTES) {
         hasAllEfficientImages = false;
       }
 
+      let jpegSavings;
       if (/(jpeg|bmp)/.test(image.mimeType)) {
-        const jpegSavings = UsesOptimizedImages.computeSavings(image, 'jpeg');
+        jpegSavings = UsesOptimizedImages.computeSavings(image, 'jpeg');
         if (jpegSavings.bytes > 0) {
           hasAllEfficientImages = false;
-          label += `, jpeg savings: ${jpegSavings.percent}%`;
         }
       }
 
       totalWastedBytes += webpSavings.bytes;
-      results.push({url, label});
+      results.push({
+        url,
+        total: `${originalKb} KB`,
+        webpSavings: `${webpSavings ? webpSavings.percent + '%' : ''}`,
+        jpegSavings: `${jpegSavings ? jpegSavings.percent + '%' : ''}`
```

Comment:
either add a check for it to be positive here or move `jpegSavings = ...` inside the if above?